code/backtracking/src/all_palindrome_partitions_of_a_string/PalindromePartitionsOfString.cpp

```diff
@@ -0,0 +1,61 @@
+#include <bits/stdc++.h> 
+using namespace std; 
+  
+bool checkPalindrome(string str) { 
```

Comment:
Better ways to check this without as much code; check our implementation in the respective folder.